tests/test_writers.py

```diff
@@ -191,3 +192,69 @@ def test_write_sssom_ontoportal_json(self):
             self.mapping_count,
             f"{path} has the wrong number of mappings.",
         )
+
+    # TODO: Determine assertions that should pass
+    # TODO: Fix failing assertions that should pass
+    # TODO: before merging, remove pass/fail comments at end of assertion lines
+    def test_missing_entries(self):
```

Comment:
## Which assertions are appropriate?
- [ ] @matentzn review

Can you look at the assertions I've made and determine which ones you think should be expected to pass, and which should not?

I've left temporary comments next to each assertion showing their current pass/fail status.